x-pack/test/functional/apps/monitoring/kibana/instances.js

```diff
@@ -33,9 +33,9 @@ export default function ({ getService, getPageObjects }) {
     it('Kibana Cluster Summary Status shows correct info', async () => {
       expect(await kibanaClusterSummaryStatus.getContent()).to.eql({
         instances: '1',
-        memory: '219.7 MB / 1.4 GB',
-        requests: '172',
-        connections: '162',
+        memory: '219.6 MB / 1.4 GB',
+        requests: '174',
+        connections: '174',
```

Comment:
This comes from changing the metric used in `x-pack/plugins/monitoring/server/lib/kibana/get_kibanas_for_clusters.js`

The `metric` passed to `createQuery` was `ElasticsearchMetric`: `{"uuidField":"source_node.uuid","timestampField":"timestamp"}`

now it is `KibanaClusterMetric`: `{"uuidField":"cluster_uuid","timestampField":"kibana_stats.timestamp"}`

`uuidField` in the metric isn't used since there isn't a `uuid` field passed to `createQuery`. The change in the `timestampField` is what accounts for this difference.